pom.xml

```diff
@@ -62,6 +62,7 @@
         <dep.mockito.version>5.8.0</dep.mockito.version>
         <dep.mysqlconnector.version>8.0.17</dep.mysqlconnector.version>
         <dep.okhttp.version>3.9.0</dep.okhttp.version>
+        <dep.oracle.version>21.9.0.0</dep.oracle.version>
```

Comment:
It would be nice to use LTS version like https://github.com/trinodb/trino/pull/19851

Also, please inline this property since it's used from single place.

Reply:
Agreed with @ebyhr ..